### DISCUSSION TITLE
Duda con output de Test 1 para parte de heap market

### DISCUSSION THREAD

**Discussion by try-except** [IIC2133-PUC/2024-1#124] [Tarea 1]
Hola! Tengo una duda en este `correct-output.txt` del primer testcase de la parte de heap market:
![image](https://github.com/IIC2133-PUC/2024-1/assets/69283830/fc1fce32-5b5b-42ae-bcba-d7adf61a122d)
Me da la sensación que en el reporte de estado de mercado la orden de venta `12 por 80` no debería aparecer porque ya se ejecutó en la línea 82. Quizás entendí algo mal del enunciado pero creo que esa orden debería haberse eliminado del mercado al ejecutarse

**Comment by sebacarrascop** [comment]
hola!, nada dice que un mismo usuario no puede volver a hacer una misma orden de compra o venta, incluso puede ser que sean seguidas

**Reply by try-except** [reply]
Es cierto pero esa orden de venta se registra una única vez
![image](https://github.com/IIC2133-PUC/2024-1/assets/69283830/6f5b6895-91d9-4c4b-92fb-1886dfd28dda)

**Reply by try-except** [reply]
Ya pillé el error, el tema es que el usuario 80 vende a monto 12 en la línea 41, pero probablemente en algún punto abre otra oferta de venta y en la línea 82 lo que se ejecuta es una compra con monto 12, por eso pareciera que el output está confundido.
